src/index.js

```diff
@@ -1,41 +0,0 @@
-import './style.css';
-import reload from './reload.svg';
-import enter from './enter.svg';
-
-import {
-  getmylist, addmylist, editmylist, deletemylist,
-} from './functionality.js';
-import { changeTodoStatus, DeleteUncompleted, DeleteCompleted } from './TaskStatus.js';
-
-const reloadIcon = document.getElementById('reload-img');
-const enterIcon = document.getElementById('enter-icon');
-enterIcon.src = enter;
-reloadIcon.src = reload;
-
-const listGroup = document.querySelector('.list-container');
-const newTask = document.querySelector('.add-form').querySelector('input');
```

Comment:
newTask has already been declared at functionality.js, it would be better to import it instead of declaring it again.